gulp/deployment.js

```diff
@@ -614,11 +612,7 @@ async function deployDApps(externals, version) {
           runtime.nameResolver.ensContract, 'owner', runtime.nameResolver.namehash(checkAddress));
         if (owner === '0x0000000000000000000000000000000000000000') {
           try {
-            if (splitEns.length -1 === i) {
-              await runtime.nameResolver.claimPermanentAddress(checkAddress, deploymentAccount);
-            } else {
-              await runtime.nameResolver.setAddress(checkAddress, '0x0000000000000000000000000000000000000000', deploymentAccount);
-            }
+            await runtime.nameResolver.claimPermanentAddress(checkAddress, deploymentAccount);
           } catch (ex) {
             console.log(ex);
             // claim permanent address is only needed for root addresses
```

Comment:
consider checking on the specified network (mainnet uses permanent address, testnet only setAddress and not trying first permanent address and then if this throws use set address